concepts/basics/about.md

```diff
@@ -0,0 +1,108 @@
+# About
+
+[Lua](https://lua.org) is a simple yet powerful, efficient, lightweight, embeddable scripting language. It is designed, implemented, and maintained at [PUC-RIO](https://www.puc-rio.br/) in Brazil since 1993 by the team of Roberto Ierusalimschy, Waldemar Celes and Luiz Henrique de Figueiredo.
+
+## Comments
+
+Singleline comments in Lua start with `--` and multiline comments start with `--[[` and end with `]]`.
```

Comment:
```suggestion
Single line comments in Lua start with `--` and multiline comments start with `--[[` and end with `]]`.
```

As far as I can tell, it's either "single line" or "single-line" but "multiline" or "multi-line".